### PR TITLE
Reset password

### PR DESCRIPTION
**Pull request by PedroCarrillo**

### CODE REVIEW

File: app/src/main/java/com/belatrixsf/allstars/networking/retrofit/responses/ResetPasswordResponse.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * Created by icerrate on 19/05/16.
+ */
+public class ResetPasswordResponse {
+
+    private String data;
+
+    public String getData() {
+        return data;
+    }
+}
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/networking/retrofit/responses/ResetPasswordResponse.java]: unused

File: app/src/main/java/com/belatrixsf/allstars/ui/resetpassword/ResetPasswordPresenter.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+import javax.inject.Inject;
+
+/**
+ * Created by icerrate on 19/05/16.
+ */
+public class ResetPasswordPresenter extends AllStarsPresenter<ResetPasswordView> {
+
+    static final int MIN_PASSWORD_LENGHT = 4;
+
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/resetpassword/ResetPasswordPresenter.java]: length

File: app/src/main/java/com/belatrixsf/allstars/ui/resetpassword/ResetPasswordPresenter.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import com.belatrixsf.allstars.utils.ServiceError;
+
+import javax.inject.Inject;
+
+/**
+ * Created by icerrate on 19/05/16.
+ */
+public class ResetPasswordPresenter extends AllStarsPresenter<ResetPasswordView> {
+
+    static final int MIN_PASSWORD_LENGTH = 4;
```
Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/ui/resetpassword/ResetPasswordPresenter.java]: should be an application constant

File: app/src/main/java/com/belatrixsf/allstars/ui/resetpassword/ResetPasswordPresenter.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+    @Inject
+    public ResetPasswordPresenter(ResetPasswordView view, EmployeeManager employeeManager) {
+        super(view);
+        this.employeeManager = employeeManager;
+    }
+
+    public void checkIfInputsAreValid(String oldPassword, String newPassword, String repeatNewPassword) {
+        boolean valid = false;
+        if (!oldPassword.isEmpty() && !newPassword.isEmpty() && !repeatNewPassword.isEmpty()){
```
Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/ui/resetpassword/ResetPasswordPresenter.java]: follow the format please, add a space after the if and before the opening curly brace 
if (condition) {

**Pull request merged**